examples/distributedLock/README.md

```diff
@@ -0,0 +1,87 @@
+# Example - Distributed Lock APIs
+
+This example demonstrates the Distributed Lock APIs in Dapr.
```

Comment:
Maybe rework this to:


This example demonstrates 2 features (Try Lock & Unlock) from the [Distributed lock API from Dapr](https://github.com/dapr/dapr/issues/3549) that helps developers to keep their data safe from race conditions. For more information, check out the documentation at <insert link>